src/components/OnePerson.tsx

```diff
@@ -0,0 +1,71 @@
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  searchParams: URLSearchParams;
+};
+
+export const OnePerson: React.FC<Props> = ({
+  person,
+  people,
+  searchParams,
+}) => {
+  const { personSlug } = useParams();
+  const selectedPerson = personSlug ? personSlug : '';
+
+  function findParentLink(name: string | null) {
+    if (name === null) {
+      return '-';
+    }
+
+    const parent = people.find(p => p.name === name);
+
+    if (parent && parent.sex === 'f') {
```

Comment:
Create a enum for 'm', 'f' and use it everywhere

Reply:
<img width="671" alt="image" src="https://github.com/user-attachments/assets/152ba63c-728c-40ae-aa40-a5f2d3e85cf6">